src/turndown_test.js

```diff
@@ -311,4 +311,105 @@ describe("turndown", () => {
+      </fieldset></div>
+      <div class="action"><button id="Q2_button" onclick="checkAnswer({2: 'multiple_choice'})" class="problem_mo_button">Check</button><button id="Q2_button_show" onclick="showHideVidSolution({2: 'multiple_choice'}, 2, [2])" class="problem_mo_button">Show Answer Videos</button></div>
+      <div id="S2_div" class="problem_solution" tabindex="-1">
+      <p>15286813pythonvs.math14606776</p>
+      </div>
+    `
+    const markdown = await html2markdown(inputHTML)
+    assert.equal(
+      markdown,
+      `Which is allowed in Python?\n\n{{< multiple-choice-question questionId="Q2_div">}}` +
+        `{{<choices>}}{{<choice isCorrect="false">}}&nbsp;x + y = 2&nbsp;{{</choice>}}\n` +
+        `{{<choice isCorrect="false">}}&nbsp;x\\*x = 2&nbsp;{{</choice>}}\n` +
+        `{{<choice isCorrect="false">}}&nbsp;2 = x&nbsp;{{</choice>}}\n` +
+        `{{<choice isCorrect="true">}}&nbsp;xy = 2&nbsp;{{</choice>}}\n` +
+        `{{<choice isCorrect="false">}}&nbsp;None of the Above&nbsp;{{</choice>}}` +
+        `{{</choices>}}\n` +
+        `{{<solution>}}15286813pythonvs.math14606776{{</solution>}}` +
+        `{{< /multiple-choice-question >}}`
+    )
+  })
+
+  it("parses multiple choice widget questions", async () => {
+    const inputHTML = `
+      <script type="text/javascript">
+
+        $( function($){ 
+          var quizMulti = {
+            multiList: [
+              {
+                ques: "If you compare the elasticity of short-run supply in the markets for two different goods and one market has more firms than the other, which will have a more elastic supply curve?",
+                ans: "The market with more firms.",
+                ansSel: ["The market with fewer firms.", "There is no difference.", "It depends on the specific production function."],
+                ansInfo: "The supply curve becomes flatter (more elastic) with more firms in the market, because a given increase in price calls forth more production when there are many firms rather than one."
+              },
+              {
+                ques: "In the long run, firms should decide to shut down if what condition holds?",
+                ans: "Price is less than both average cost and average variable cost.",
+                ansSel: ["Price is less than average variable cost.", "Price is less than average cost.", "Price is less than average fixed cost."],
+                ansInfo: "In the long run, all costs are variable, and thus average variable cost and average cost are equivalent. The firm will shut down if price is less than average cost. Average fixed cost is not a relevant concept in the long-run, because all costs are considered to be variable."
+              }
+            ]
+          };
+          var options = {
```

Comment:
Do we need to do any parsing of these options that seem to appear on each of the quizzes?  Things like `animationType` probably don't matter, but I'm wondering if the `title` property needs to be displayed somewhere or if any of these other options need to be taken into account.  Feel free to make a separate issue for this, as we may not have all the requirements at the moment.  I figured I would mention it though because they do seem to appear on every quiz and are being passed into this "jQuizMe" library.  If they change the behavior we should note that and potentially implement whatever the options are doing.

Reply:
The title is not actually displayed anywhere is original ocw. Here is the page where the example was taken from:
https://ocw.mit.edu/courses/economics/14-01sc-principles-of-microeconomics-fall-2011/unit-3-producer-theory/competition-ii/

The rest of the options are settings for the quiz widget. We aren't actually going to display the widget in ocw-next, just show the questions in the page.